src/components/RGBColorPicker/RGBColorPicker.css

```diff
@@ -0,0 +1,3 @@
+.rgb {
+    margin: 30px 0 50px 0;
```

Comment:
Esto sería lo mismo:
```suggestion
    margin: 30px 0 50px;
```